src/SpecificDissipationRateEquationSystem.C

```diff
@@ -269,8 +272,12 @@ SpecificDissipationRateEquationSystem::register_interior_algorithm(
         if (!elementMassAlg)
           nodeAlg.add_kernel<ScalarMassBDFNodeKernel>(realm_.bulk_data(), sdr_);
 
-        if (TurbulenceModel::SST == realm_.solutionOptions_->turbulenceModel_) {
+        if ( TurbulenceModel::SST == realm_.solutionOptions_->turbulenceModel_ && 
```

Comment:
For clarity I think I would prefer nesting the if conditions. 
If SST
   if gamma active
   else
else if SSTLR

Reply:
I tried this, but it actually made the implementations very lengthy. Let's discuss this during the meeting this afternoon.

Reply:
Modifications made. Done.